markdown/tracing-ea/tracing-ea.md

```diff
@@ -0,0 +1,380 @@
+author: pl89
+summary:
+id: tracing-ea
+tags:
+categories:
```

Comment:
FYI, the first item in the list of "categories" will decide what image is showed on the codelab once it is public. I would suggest either "Opentelemetry" or "Solace" for this one.

Reply:
Thanks, I will fix that.